specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/SpringMiddlewares.json

```diff
@@ -0,0 +1,925 @@
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigServer"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "EurekaServerResourceCollection": {
+      "description": "Azure Spring Apps Eureka Server collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EurekaServer"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ConfigServerSettings": {
+      "description": "The settings of config server.",
+      "type": "object",
+      "properties": {
+        "gitProperty": {
+          "$ref": "#/definitions/ConfigServerGitProperty",
+          "description": "Property of git environment."
+        }
+      }
+    },
+    "ConfigServerGitProperty": {
+      "description": "Property of git.",
+      "required": [
+        "uri"
+      ],
+      "type": "object",
+      "properties": {
+        "repositories": {
+          "description": "Repositories of git.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitPatternRepository"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "uri": {
+          "description": "URI of the repository",
+          "type": "string"
+        },
+        "label": {
+          "description": "Label of the repository",
+          "type": "string"
+        },
+        "searchPaths": {
+          "description": "Searching path of the repository",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "username": {
+          "description": "Username of git repository basic auth.",
+          "type": "string"
+        },
+        "password": {
+          "description": "Password of git repository basic auth.",
+          "type": "string"
+        },
+        "hostKey": {
+          "description": "Public sshKey of git repository.",
+          "type": "string"
+        },
+        "hostKeyAlgorithm": {
+          "description": "SshKey algorithm of git repository.",
+          "type": "string"
+        },
+        "privateKey": {
+          "description": "Private sshKey algorithm of git repository.",
+          "type": "string"
+        },
+        "strictHostKeyChecking": {
```

Comment:
>strictHostKeyChecking [](http://example.com/codeflow?start=9&length=21)

This could be defined as: hostKeyCheckMode and values as "strict", "none".